doc/man/8/osdmaptool.rst

```diff
@@ -61,6 +70,77 @@ To view the result::
 
         osdmaptool --print osdmap
 
+To view the mappings of placement groups for pool 0::
+
+        osdmaptool --test-map-pgs-dump rbd --pool 0
+
+        pool 0 pg_num 8
+        0.0     [0,2,1] 0
+        0.1     [2,0,1] 2
+        0.2     [0,1,2] 0
+        0.3     [2,0,1] 2
+        0.4     [0,2,1] 0
+        0.5     [0,2,1] 0
+        0.6     [0,1,2] 0
+        0.7     [1,0,2] 1
+        #osd    count   first   primary c wt    wt
+        osd.0   8       5       5       1       1
+        osd.1   8       1       1       1       1
+        osd.2   8       2       2       1       1
+         in 3
+         avg 8 stddev 0 (0x) (expected 2.3094 0.288675x))
+         min osd.0 8
+         max osd.0 8
+        size 0  0
+        size 1  0
+        size 2  0
+        size 3  8
+
+In which,
+ #. pool 0 has 8 placement groups. And two tables follow:
+ #. A table for placement groups. Each row presents a placement group. With columns of:
+
+    * placement group id,
+    * acting set, and
+    * primary OSD.
+ #. A table for all OSDs. Each row presents an OSD. With columns of:
+
+    * count of placement groups being mapped to this OSD,
+    * count of placement groups where this OSD is the first one in their acting sets,
+    * count of placement groups where this OSD is the primary of them,
+    * the CRUSH weight of this OSD, and
+    * the weight of this OSD.
+ #. Looking at the number of placement groups held by 3 OSDs. We have
+
+    * avarge, stddev, stddev/average, expected stddev, expected stddev / average
+    * min and max
+ #. The number of placement groups mapping to n OSDs. In this case, we have 8 placement
+    groups sizing 3 replicas.
+
+In a less-balanced cluster, we could have following output for the statistics of
+placement group distribution, whose std deviation is 1.41421::
```

Comment:
@tchaikov  You might want to change  std to standard, which looks more formal and better.